### PR TITLE
fix(common): solve Select or unselect all rows at once

### PR DESCRIPTION
**Pull request by aziz-access**
**What is the current behavior?** (You can also link to an open issue here)

this behavior related to #1512 

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

**Comment by alecarn** [comment]
@pelord ready

File: packages/common/src/lib/entity/entity-table/entity-table.component.ts [hunk 1]
```diff
@@ -604,13 +604,11 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
...
-    this.store.state.updateAll({ selected: toggle });
-    if (toggle === true) {
-      const entities = this.store.stateView
-        .all()
-        .map((record: EntityRecord<object>) => record.entity);
-      this.entitySelectChange.emit({ added: [entities] });
-    }
+    const entities = this.store.stateView
+      .all()
+      .map((record: EntityRecord<object>) => record.entity);
```
[the remark this answers is not in the document]
Comment (aziz-access) [reply packages/common/src/lib/entity/entity-table/entity-table.component.ts:609]: > Semble être l'équivalent: `this.store.view.all()`? On n'aurait pas besoin de faire une itération pour aller rechercher les entités

bien vu!

**Review by pelord — COMMENTED** [review]
Validate the behavior of spatial filter

File: packages/common/src/lib/entity/entity-table/entity-table.component.ts [hunk 2]
```diff
@@ -604,13 +604,8 @@ export class EntityTableComponent implements OnInit, OnChanges, OnDestroy {
       return;
     }
 
-    this.store.state.updateAll({ selected: toggle });
-    if (toggle === true) {
-      const entities = this.store.stateView
-        .all()
-        .map((record: EntityRecord<object>) => record.entity);
-      this.entitySelectChange.emit({ added: [entities] });
```
[the remark this answers is not in the document]
Comment (pelord) [reply packages/common/src/lib/entity/entity-table/entity-table.component.ts:612]: review the comment please

image

**Review by alecarn — APPROVED** [review]

**Pull request merged**